CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 [PyPI History][1]
 
 [1]: https://pypi.org/project/demisto-py/#history
+## 2.0.21
```

Comment:
```suggestion
## 2.0.22
```

Current version right now is 2.0.21 due to a minor issue with the previous release.